databrary/datavyu-ffmpegplugin
# Cleanup testing

- Add hook for initialization and add test for it
- Split tests for mpv/ffmpeg
- MediaPlayerTest contains the high-level test logic
- Mpv/Ffmpeg...Test contains the data providers and wiring of providers
to tests
- MediaPlayerSync (not a good name yet) contains the locks to ensure to
wait for state changes
